File: java/src/main/java/com/gaston/macbook/simplemvp/presentation/show_apod/model/Apod.java
package com.gaston.macbook.simplemvp.presentation.show_apod.model;

import com.google.gson.annotations.SerializedName;

public class Apod {

    private String explanation;
    private String hdurl;
    @SerializedName("url")
    private String lowresurl;
    private String title;
    private String date;
    private String copyright;

    public Apod(String explanation, String hdurl, String lowresurl, String title) {
        this.explanation = explanation;
        this.hdurl = hdurl;
        this.lowresurl = lowresurl;
        this.title = title;
    }

    public String getDate() {
        return date;
    }

    public String getCopyright() {
        return copyright;
    }

    public String getExplanation() {
        return explanation;
    }

    public void setExplanation(String explanation) {
        this.explanation = explanation;
    }

    public String getHdurl() {
        return hdurl;
    }

    public String getLowresurl() {
        return lowresurl;
    }

    public String getTitle() {
        return title;
    }

}

File: kotlin/src/main/java/com/gaston/macbook/kotlinsimplemvp/presentation/show_apod/model/Apod.kt
package com.gaston.macbook.kotlinsimplemvp.presentation.show_apod.model

import com.google.gson.annotations.SerializedName

data class Apod(val explanation: String = "",
                val hdurl: String = "",
                @SerializedName("url") val lowresurl: String = "",
                val title: String = "",
                val date: String = "",
                val copyright: String = "")